### DISCUSSION TITLE
How to make grid item size fixed?

### DISCUSSION THREAD

**Discussion by scarf005** [clauderic/dnd-kit#919] [Q&A]
https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/?path=/story/presets-sortable-grid--variable-sizes

when i move 2x2 sized components, their size gets shrinked into 1x1. the one getting replaced is enlarged into 2x2.
How could i make each components retain their size?

**Comment by little077** [comment]
这么久了都没有一个案例么，这实在是太糟糕了
